lua/harpoon/tmux.lua

```diff
@@ -0,0 +1,210 @@
+local harpoon = require("harpoon")
+local log = require("harpoon.dev").log
+local global_config = harpoon.get_global_settings()
+local utils = require("harpoon.utils")
+
+local M = {}
+local tmux_windows = {}
+
+if global_config.tmux_autoclose_windows then
+    vim.cmd([[
+        augroup HARPOON_TMUX
+        autocmd!
+        autocmd VimLeave * :lua require('harpoon.tmux').clear_all()
+    ]])
+end
+
+local function create_terminal()
+    log.trace("tmux: _create_terminal())")
+
+    local window_id
+
+    -- Create a new tmux window and store the window id
+    local out, ret, _ = utils.get_os_command_output({
+        "tmux",
+        "new-window",
+        "-P",
+        "-F",
+        "#{pane_id}",
```

Comment:
@pranavrao145 shouldn't this be window_id instead ?

I notice that the go to terminal window command mostly works, however if i start creating panes, the go to terminal command starts becoming flaky